src/test/java/org/openmrs/contrib/qaframework/automation/EditPatientRelationshipSteps.java

```diff
@@ -12,9 +18,12 @@
 import io.cucumber.java.en.When;
 
 public class EditPatientRelationshipSteps extends Steps {
+	private static final By NEXT_BUTTON = By.id("next-button");
 	private FindPatientPage findpatientPage;
+	private String patientDashboardId;
+	private EditPatientRelationshipPage editPatientRelationshipPage;
 
-	private ClinicianFacingPatientDashboardPage clinicianFacingPatientDashboardPage;
+	private ClinicianFacingPatientDashboardPage dashboardPage;
```

Comment:
any reason for renaming this?

Reply:
For simplicity but we can as well revert it back

Reply:
the former name is more clearer

Reply:
Let me revert that thanks